### PR TITLE
Add A006880 Number of primes sequence

### PR DESCRIPTION
**Pull request by OleksiiHe**
Adding A006880 Number of primes sequence
[OEIS](https://oeis.org/A006880)
[Wikipedia](https://wikipedia.org/wiki/Prime-counting_function)

- [x] I have performed a self-review of my code
- [x] My code follows the style guidelines of this project
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Comments in areas I changed are up to date
- [x] I have added comments to hard-to-understand areas of my code
- [x] I have made corresponding changes to the README.md

### CODE REVIEW

**Review by siriak — CHANGES_REQUESTED** [review]
You're keeping track of primes by yourself, but could use the existing primes sequence https://github.com/TheAlgorithms/C-Sharp/blob/master/Algorithms/Sequences/PrimesSequence.cs

File: Algorithms/Sequences/NumberOfPrimesByPowersOf10Sequence.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    {
+        /// <summary>
+        /// Gets sequence of numbers of primes.
+        /// </summary>
+        public IEnumerable<BigInteger> Sequence
+        {
+            get
+            {
+                ISequence primes = new PrimesSequence();
+                var prevPowerOf10 = new BigInteger(0);
```
Comment (siriak) [inline Algorithms/Sequences/NumberOfPrimesByPowersOf10Sequence.cs:28]: 0 is not a power of 10

File: Algorithms/Sequences/NumberOfPrimesByPowersOf10Sequence.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                var powerOf10 = new BigInteger(1);
+                var counter = new BigInteger(0);
+
+                while (true)
+                {
+                    counter += primes.Sequence.
+                        SkipWhile(p => p < prevPowerOf10).TakeWhile(p => p < powerOf10).Count();
```
Comment (siriak) [inline Algorithms/Sequences/NumberOfPrimesByPowersOf10Sequence.cs:35]: Please refactor this to iterate over primes only once instead of on every iteration

**Review by siriak — CHANGES_REQUESTED** [review]

File: Algorithms/Sequences/NumberOfPrimesByPowersOf10Sequence.cs (hunk 2 above)
Comment (OleksiiHe) [reply Algorithms/Sequences/NumberOfPrimesByPowersOf10Sequence.cs]: Refactoring has been completed.

File: Algorithms/Sequences/NumberOfPrimesByPowersOf10Sequence.cs (hunk 1 above)
Comment (OleksiiHe) [reply Algorithms/Sequences/NumberOfPrimesByPowersOf10Sequence.cs]: Variable has been removed during refactoring.

**Review by siriak — APPROVED** [review]
LGTM, thanks!

**Pull request merged**